packages/paste-website/src/pages/components/table/index.mdx

```diff
@@ -0,0 +1,436 @@
+
+<LivePreview scope={{Table, THead, TBody, Tr, Th, Td}} language="jsx">
+  {`<Table>
+  <THead>
+    <Tr>
+      <Th width="size20">First Name</Th>
+      <Th>Last Name</Th>
+    </Tr>
+  </THead>
+  <TBody>
+    <Tr>
+      <Td>Tamir</Td>
+      <Td>Rice</Td>
+    </Tr>
+    <Tr>
+      <Td>Philando</Td>
+      <Td>Castile</Td>
+    </Tr>
+    <Tr>
+      <Td>Trayvon</Td>
+      <Td>Martin</Td>
+    </Tr>
+    <Tr>
+      <Td>Breonna</Td>
+      <Td>Taylor</Td>
+    </Tr>
+    <Tr>
+      <Td>Shantel</Td>
+      <Td>Davis</Td>
+    </Tr>
+    <Tr>
+      <Td>Walter</Td>
+      <Td>Scott</Td>
+    </Tr>
+  </TBody>
+</Table>`}
+</LivePreview>
+
+### Column Alignment
+
+Table cells (`Td` or `Th`) can be set with either left, center, or right text alignment. By default they are left-aligned.
+
+<LivePreview scope={{Table, THead, TBody, Tr, Th, Td}} language="jsx">
+  {`<Table>
+  <THead>
+    <Tr>
+      <Th>First Name</Th>
+      <Th alignment="right">Last Name</Th>
+    </Tr>
+  </THead>
+  <TBody>
+    <Tr>
+      <Td>Tamir</Td>
+      <Td alignment="right">Rice</Td>
+    </Tr>
+    <Tr>
+      <Td>Philando</Td>
+      <Td alignment="right">Castile</Td>
+    </Tr>
+    <Tr>
+      <Td>Trayvon</Td>
+      <Td alignment="right">Martin</Td>
+    </Tr>
+    <Tr>
+      <Td>Breonna</Td>
+      <Td alignment="right">Taylor</Td>
+    </Tr>
+    <Tr>
+      <Td>Shantel</Td>
+      <Td alignment="right">Davis</Td>
+    </Tr>
+    <Tr>
+      <Td>Walter</Td>
+      <Td alignment="right">Scott</Td>
+    </Tr>
+  </TBody>
+</Table>`}
+</LivePreview>
+
+### Row Alignment
+
+Table rows (`Tr`) can be set with either top, middle, or bottom vertical alignment. By default they are middle-aligned.
+
+<LivePreview scope={{Table, THead, TBody, Tr, Th, Td}} language="jsx">
+  {`<Table>
+  <THead>
+    <Tr>
+      <Th width="size20">Author</Th>
+      <Th>Quote</Th>
+    </Tr>
+  </THead>
+  <TBody>
+    <Tr verticalAlignment="top">
+      <Td>Chinua Achebe</Td>
+      <Td>The impatient idealist says: ‘Give me a place to stand and I shall move the earth.’ But such a place does not exist. We all have to stand on the earth itself and go with her at her pace.</Td>
+    </Tr>
+    <Tr verticalAlignment="top">
+      <Td>James Baldwin</Td>
+      <Td>Anyone who has ever struggled with poverty knows how extremely expensive it is to be poor.</Td>
+    </Tr>
+    <Tr verticalAlignment="top">
+      <Td>Dorothy West</Td>
+      <Td>To know how much there is to know is the beginning of learning to live.</Td>
+    </Tr>
+    <Tr verticalAlignment="top">
+      <Td>W.E.B. Du Bois</Td>
+      <Td>Herein lies the tragedy of the age: not that men are poor—all men know something of poverty; not that men are wicked—who is good? Not that men are ignorant—what is truth? Nay, but that men know so little of men.</Td>
+    </Tr>
+  </TBody>
+</Table>`}
+</LivePreview>
+
+## Anatomy
```

Comment:
Do you want to include THead in this section, too?

Reply:
The `THead` is just a box `as="thead"`, so theres nothing really to mention on it. We're not doing any other custom styling for it.